alt_src/alt_src.py

```diff
@@ -2,10 +2,12 @@
 '''
 Given an srpm and product, stage for alt-src release
 '''
+from __future__ import print_function
 
-import ConfigParser
+import six
```

Comment:
It looks like the imports were previously in alphabetical order. Could you please do the same for readability?

Reply:
Fixed.